common/doc.go

```diff
@@ -18,5 +18,24 @@
 // This file is derived from common/bytes.go (2018/06/04).
 // Modified and improved for the klaytn development.
 
-// Package common contains various helper functions.
+/*
+Package common contains various helper functions.
+
+The `common` package defines commonly used data types and constants.
+
+And also it provides cache and helper functions.
```

Comment:
Please consider merging above three sentences into one like below =)

Package common contains various helper functions, commonly used data types and constants.

Reply:
@ehnuje Good suggestion. Thanks!